iterative_verifier.go

```diff
@@ -572,6 +573,27 @@ func (v *IterativeVerifier) tableIsIgnored(table *schema.Table) bool {
 	return false
 }
 
+func (v *IterativeVerifier) columnsToVerify(table *schema.Table) []schema.TableColumn {
+	ignoredCols, containsIgnoredColumns := v.IgnoredColumns[table.Name]
+	if !containsIgnoredColumns {
+		return table.Columns
+	}
+
+	ignoredColsSet := make(map[string]struct{})
```

Comment:
Would it be possible to structure the configuration of the ignored columns to remove lines 582-585? Or, instead of having to iterate and create the set using a map each time would it be better to just do it at startup instead of this phase in the runtime?